docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +166,43 @@ $menu->reorderChildren(
 );
 ```
 
+## Dashboard
+
+To extend the Dashboard, make use of an event subscriber.
+
+In the following example, the `DasboardEventSubscriber.php` reverses the order of sections of the Dashboard
```

Comment:
Is `Sections` a domain-related thing? If so, could we first upper-case this word?

Reply:
No, it's not, it's unrelated to "Sections" as in Admin Panel